services/traction/api/endpoints/dependencies/tenant_context.py

```diff
@@ -0,0 +1,13 @@
+from starlette_context import context
+from starlette.exceptions import HTTPException
+from starlette import status
+
+
+def get_from_context(name: str):
+    result = context.get(name)
+    if not result:
```

Comment:
i don't think throwing an HTTP exception when querying the database level is appropriate.